src/vfc_precexp

```diff
@@ -0,0 +1,514 @@
+#!/usr/bin/env python3
+import numpy as np
+import pandas as pd 
+import matplotlib.pyplot as plt
+import random
+import math
+import sys
+import struct
+import shlex
+import os.path
+import subprocess
+import signal
+from os import listdir
+
+vfc_profile_file = "vfc_profile.txt"
+vfc_config_file = "vfc_config.txt"
+output_dir = ["vfc_ref", "vfc_std"]
+
+def set_environment_variable(envVar, value, env):
+	#replace current value, if the environment variable already exists
```

Comment:
These would be nicer as docstrings

Reply:
I agree, I changed that and it is now a docstring.